pkg/datapath/loader/netlink.go

```diff
@@ -220,7 +220,7 @@ func setupVethPair(name, peerName string) error {
 // the first step of datapath initialization, then performs the setup (and
 // creation, if needed) of those interfaces. It returns two links and an error.
 // By default, it sets up the veth pair - cilium_host and cilium_net.
-func setupBaseDevice(nativeDevs []netlink.Link, mode baseDeviceMode, mtu int) (netlink.Link, netlink.Link, error) {
+func setupBaseDevice(mode baseDeviceMode, mtu int) (netlink.Link, netlink.Link, error) {
```

Comment:
Looks like `mode` is unused as well and can be removed

```suggestion
func setupBaseDevice(mtu int) (netlink.Link, netlink.Link, error) {
```

Reply:
got it